index.js

```diff
@@ -439,6 +447,12 @@ Converter.prototype.buildParameter = function(oldParameter) {
       'Complex type is used inside non-body argument.');
   }
 
+  //According to Swagger 2.0 spec: If the parameter is in "path",
+  //this property is required and its value MUST be true.
+  if (parameter.in === 'path') {
```

Comment:
@webron do you have to specify `required` for path parameters? I see schema validates a spec that has a path parameter without `required` property. 
@IvanGoncharov if it's optional, it's probably a good idea to explicitly specify it here.